### PR TITLE
support log desensitization

### PR DESCRIPTION
**Pull request by marsishandsome**
### What problem does this PR solve? 
close https://github.com/pingcap/tispark/issues/1735

### What is changed and how it works?
provide a system environment configuration `TiSparkLogDesensitizationLevel`

`TiSparkLogDesensitizationLevel=1` disable LogDesensitization, otherwise enable LogDesensitization (default enable LogDesensitization)

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/util/LogDesensitization.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+
+package com.pingcap.tikv.util;
+
+public class LogDesensitization {
+  private static boolean enableLogDesensitization = getLogDesensitization();
+
+  public static String hide(String info) {
+    if (enableLogDesensitization) {
+      return "[hide]";
```
Comment (birdstorm) [inline tikv-client/src/main/java/com/pingcap/tikv/util/LogDesensitization.java:23]: How about ***？

**Review by birdstorm — APPROVED** [review]
lgtm

File: tikv-client/src/main/java/com/pingcap/tikv/util/LogDesensitization.java (hunk 1 above)
Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/util/LogDesensitization.java]: good idea

**Pull request merged**